src/Components/AffectedClustersTable/AffectedClustersTable.spec.ct.js

```diff
@@ -57,11 +62,26 @@ data.forEach(
 // default sorting
 data = _.orderBy(data, ['last_checked_at'], ['desc']);
 
-function filterData(text = '') {
-  return _.filter(data, (it) =>
-    it.name.toLowerCase().includes(text.toLowerCase())
-  );
-}
+const filtersConf = {
+  name: {
+    selectorText: 'Name',
+    values: SEARCH_ITEMS,
+    type: 'input',
+    filterFunc: (it, value) =>
+      it.name.toLowerCase().includes(value.toLowerCase()),
+  },
+  version: {
+    selectorText: 'Version',
+    values: VERSION_COMBINATIONS,
```

Comment:
`VERSION_COMBINATIONS` contains nested arrays: https://github.com/RedHatInsights/ocp-advisor-frontend/blob/fa3793e788361d0cb2cd95962c5196257d24988f/cypress/utils/filters.js#L13. To get only the possible values, you have to flatten the array and remove duplicates

Reply:
No, it is fine as it is. It is the expected behavior as it is a checkbox, so multiple values can be clicked. Have you seen something weird in the tests?

Reply:
@ikerreyes, oh, you are true, it works as expected. sorry for the noise